src/c++/perf_analyzer/request_rate_worker.h

```diff
@@ -117,9 +117,10 @@ class RequestRateWorker : public LoadWorker {
   // Returns true if the request was delayed
   bool SleepIfNecessary();
 
-  // Empty function (request rate worker doesn't need to finalize
-  // anything here)
-  void AsyncCallbackFinalize(uint32_t ctx_id) override {}
+  void AsyncCallbackFinalize(uint32_t ctx_id) override
+  {
+    total_ongoing_requests_--;
```

Comment:
I had to move the total_ongoing_reqeusts decrement from common code into concurrency_worker's callback, so I had to match that update on request_rate_worker here.

Reply:
Can this just go at the end of [LoadWorker::AsyncCallbackFuncImpl()](https://github.com/triton-inference-server/client/blob/tgerdes-refactor-requests/src/c%2B%2B/perf_analyzer/load_worker.cc#L646) after the call to `AsyncCallbackFinalize()`?

Reply:
No, because the ConcurrencyWorker implementation has `total_ongoing_requests_--` in a very specific place in AsyncCallbackFinalize(). Thus, RRW also needs to have it in AsyncCallbackFinalize().